cli.js

```diff
@@ -6,6 +6,13 @@
  */
 const {setHooksFromConfig} = require('./simple-git-hooks')
 
+const {SKIP_INSTALL_SIMPLE_GIT_HOOKS} = process.env
+
+if (['1', 'true'].includes(SKIP_INSTALL_SIMPLE_GIT_HOOKS)) {
+    console.log(`[INFO] SKIP_INSTALL_SIMPLE_GIT_HOOKS is set to "${SKIP_INSTALL_SIMPLE_GIT_HOOKS}", skipping installing hook.`)
```

Comment:
```suggestion
    console.log(`[INFO] SKIP_INSTALL_SIMPLE_GIT_HOOKS is set to "${SKIP_INSTALL_SIMPLE_GIT_HOOKS}", skipping installing hook, if you had hooks installed, you can delete them using the delete command.`)
```

@toplenboren Did you mean this? What is `delete command` by the way?

Reply:
I meant this:

![image](https://github.com/toplenboren/simple-git-hooks/assets/33755274/fa4be95a-32e8-4e02-acf3-57672518e1e1)

You can run `npm uninstall simple-git-hooks` and we will trigger `removeHooks()` or smth like that

Reply:
But that will prompt user to uninstall the package .... lets just leave it as is

Sorry for inconvinience